docs/bos/tutorial/indexer-tutorials/feed-indexer.md

```diff
@@ -16,6 +22,12 @@ The indexer `indexingLogic.js` is comprised of functions that help handle, trans
 
 A schema is also specified for the tables in which data from relevant transactions is to be persisted, this can be found in the `schema.sql` tab.
 
+:::tip
+
+This indexer can be found by [following this link](https://near.org/#/dataplatform.near/widget/QueryApi.App?selectedIndexerPath=roshaan.near/feed-indexer&view=editor-window).
```

Comment:
changing to indexer stored on dataplatform.near rather than on roshaan.near
```suggestion
This indexer can be found by [following this link](https://near.org/dataplatform.near/widget/QueryApi.App?selectedIndexerPath=dataplatform.near/social_feed&view=editor-window).
```